### PR TITLE
162 fix variable names in open prom

### PR DESCRIPTION
**Pull request by Redmonkeycloud**

### CODE REVIEW

**Review by derevirn — DISMISSED** [review]

**Comment by Redmonkeycloud** [comment]
Name convention followed: [Q/V/i][Cost/Cap/Dem/Cons/Price/etc.][more specific, if needed][Elec/Transp/Ind/Dom][etc.]
Example: If a variable calculates capacity -> QCapElecChp, VCapElecChp

**Review by derevirn — DISMISSED** [review]

File: declarations.gms [hunk 1]
```diff
@@ -272,163 +272,163 @@ qDummyObj                                     "Define dummy objective function"
 Variables
 
 *' *** Power Generation Variables
-VElecCapChpPla(allCy,CHP,YTIME)	          "Capacity of CHP Plants (GW)"
-VLoadCurveConstr(allCy,YTIME)	          "Parameter for load curve construction (1)"
-VEstBaseLoad(allCy,YTIME)	              "Estimated base load (GW)"	
-VElecDem(allCy,YTIME)                     "Total electricity demand (TWh)"
+VCapElecChp(allCy,CHP,YTIME)	          "Capacity of CHP Plants (GW)"
+VLambda(allCy,YTIME)	          "Parameter for load curve construction (1)"
```
Comment (derevirn) [inline declarations.gms:276]: There's now a VLambda. as well as a VLamda variable. This may cause errors and confusion.

Comment (Redmonkeycloud) [reply declarations.gms:276]: There is only VLambda. The VLamda parameter has been renamed.

File: declarations.gms [hunk 2]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QElecPeakLoad(allCy,YTIME)	               "Compute elerctricity peak load"		
-QBslMaxmLoad(allCy,YTIME) 	               "Compute baseload corresponding to maximum load"
-QElecBaseLoad(allCy,YTIME)	               "Compute electricity base load"
-QShrcap(allCy,PGALL,YTIME)	               "Compute SHRCAP"	
-QElecGenCap(allCy,PGALL,YTIME)	           "Compute electricity generation capacity"
+QLoadPeakElec(allCy,YTIME)	               "Compute elerctricity peak load"		
+QLoadBaseMax(allCy,YTIME) 	               "Compute baseload corresponding to maximum load"
+QLoadBaseElecExp(allCy,YTIME)	               "Compute electricity base load"
+QShareNewTechCcs(allCy,PGALL,YTIME)	               "Compute SHRCAP"	
+QCapGenPlannedPlant(allCy,PGALL,YTIME)	           "Compute electricity generation capacity"
```
Comment (giannou) [inline declarations.gms:130]: `QCapElec2`

File: declarations.gms [hunk 3]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
 QLambda(allCy,YTIME)	                   "Compute Lambda parameter"	
 QScalFacPlantDispatch(allCy,HOUR,YTIME)   "Compute the scaling factor for plant dispatching"	
-qCurrRenPot(allCy,PGRENEF,YTIME)	       "Compute current renewable potential" 	
+qPotResCurr(allCy,PGRENEF,YTIME)	       "Compute current renewable potential"
```
Comment (giannou) [inline declarations.gms:133]: `QPotRenCurr`

File: declarations.gms [hunk 4]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QElecProdPowGenPlants(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"	
-QReqElecProd(allCy,YTIME)	               "Compute total required electricity production" 	
-QElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"	
-QOverallCap(allCy,PGALL,YTIME)	           "Compute overall capacity"
-QNonChpElecProd(allCy,YTIME)	           "Compute non CHP electricity production" 	
-QAvgCapFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"	
-QNewCapYearly(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
-QElectrPeakLoad(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
-QVarCostTechnology(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
+QProdElecPowPlantsCy(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"
```
Comment (giannou) [inline declarations.gms:146]: `QProdElec`

File: declarations.gms [hunk 5]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"	
-QOverallCap(allCy,PGALL,YTIME)	           "Compute overall capacity"
-QNonChpElecProd(allCy,YTIME)	           "Compute non CHP electricity production" 	
-QAvgCapFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"	
-QNewCapYearly(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
-QElectrPeakLoad(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
-QVarCostTechnology(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
+QProdElecPowPlantsCy(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"	
+QProdReqElec(allCy,YTIME)	               "Compute total required electricity production" 	
+QGenEstElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"
```
Comment (giannou) [inline declarations.gms:148]: `QProdEstElecChp`

File: declarations.gms [hunk 6]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QNonChpElecProd(allCy,YTIME)	           "Compute non CHP electricity production" 	
-QAvgCapFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"	
-QNewCapYearly(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
-QElectrPeakLoad(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
-QVarCostTechnology(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
+QProdElecPowPlantsCy(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"	
+QProdReqElec(allCy,YTIME)	               "Compute total required electricity production" 	
+QGenEstElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"	
+QCapOverall(allCy,PGALL,YTIME)	           "Compute overall capacity"
+QProdNonChp(allCy,YTIME)	           "Compute non CHP electricity production"
```
Comment (giannou) [inline declarations.gms:150]: `QProdElecNonChp`

File: declarations.gms [hunk 7]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QAvgCapFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"	
-QNewCapYearly(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
-QElectrPeakLoad(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
-QVarCostTechnology(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
+QProdElecPowPlantsCy(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"	
+QProdReqElec(allCy,YTIME)	               "Compute total required electricity production" 	
+QGenEstElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"	
+QCapOverall(allCy,PGALL,YTIME)	           "Compute overall capacity"
+QProdNonChp(allCy,YTIME)	           "Compute non CHP electricity production" 	
+QCapAvgFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"
```
Comment (giannou) [inline declarations.gms:151]: `QCFAvgRen`

File: declarations.gms [hunk 8]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QNewCapYearly(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
-QElectrPeakLoad(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
-QVarCostTechnology(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
+QProdElecPowPlantsCy(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"	
+QProdReqElec(allCy,YTIME)	               "Compute total required electricity production" 	
+QGenEstElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"	
+QCapOverall(allCy,PGALL,YTIME)	           "Compute overall capacity"
+QProdNonChp(allCy,YTIME)	           "Compute non CHP electricity production" 	
+QCapAvgFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"	
+QCapGenNewResPlant(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"
```
Comment (giannou) [inline declarations.gms:152]: `QNewCapElec`

File: declarations.gms [hunk 9]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QVarCostTechnology(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
+QProdElecPowPlantsCy(allCy,PGALL,YTIME)   "Compute electricity production from power generation plants"	
+QProdReqElec(allCy,YTIME)	               "Compute total required electricity production" 	
+QGenEstElecChpPlants(allCy,YTIME)	               "Estimate the electricity of CHP Plants"	
+QCapOverall(allCy,PGALL,YTIME)	           "Compute overall capacity"
+QProdNonChp(allCy,YTIME)	           "Compute non CHP electricity production" 	
+QCapAvgFacRes(allCy,PGALL,YTIME)	       "Compute the average capacity factor of RES"	
+QCapGenNewResPlant(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
+QSortPlantDispatch(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
+QCostVarPlantTech(allCy,PGALL,YTIME)	   "Compute variable cost of technology"
```
Comment (giannou) [inline declarations.gms:154]: `QCostVarTechElec`

File: declarations.gms [hunk 10]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
+QCapGenNewResPlant(allCy,PGALL,YTIME)	       "Compute the new capacity added every year"	
+QSortPlantDispatch(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
+QCostVarPlantTech(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
 QElecPeakLoads(allCy,YTIME)	               "Compute Electricity peak loads"	
-QElecGenCapacity(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
-QPowPlaShaNewEquip(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
-QNewInvDecis(allCy,YTIME)	               "Compute for Power Plant new investment decision"		
-QMaxmAllowRenPotent(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
-QMnmAllowRenPot(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential" 
+QCapGenPlant(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"
```
Comment (giannou) [inline declarations.gms:156]: `QCapElec`

File: declarations.gms [hunk 11]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
+QSortPlantDispatch(allCy,PGALL,YTIME)	       "Compute Power plants sorting according to variable cost to decide the plant dispatching" 	
+QCostVarPlantTech(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
 QElecPeakLoads(allCy,YTIME)	               "Compute Electricity peak loads"	
-QElecGenCapacity(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
-QPowPlaShaNewEquip(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
-QNewInvDecis(allCy,YTIME)	               "Compute for Power Plant new investment decision"		
-QMaxmAllowRenPotent(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
-QMnmAllowRenPot(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential" 
+QCapGenPlant(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
+QShaNewEqPowPla(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"
```
Comment (giannou) [inline declarations.gms:157]: `QSharePowPlaNewEq`

File: declarations.gms [hunk 12]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
+QCostVarPlantTech(allCy,PGALL,YTIME)	   "Compute variable cost of technology" 
 QElecPeakLoads(allCy,YTIME)	               "Compute Electricity peak loads"	
-QElecGenCapacity(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
-QPowPlaShaNewEquip(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
-QNewInvDecis(allCy,YTIME)	               "Compute for Power Plant new investment decision"		
-QMaxmAllowRenPotent(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
-QMnmAllowRenPot(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential" 
+QCapGenPlant(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
+QShaNewEqPowPla(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
+QDecisNewInv(allCy,YTIME)	               "Compute for Power Plant new investment decision"
```
Comment (giannou) [inline declarations.gms:158]: `QNewInvElec`

File: declarations.gms [hunk 13]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
 QElecPeakLoads(allCy,YTIME)	               "Compute Electricity peak loads"	
-QElecGenCapacity(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
-QPowPlaShaNewEquip(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
-QNewInvDecis(allCy,YTIME)	               "Compute for Power Plant new investment decision"		
-QMaxmAllowRenPotent(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
-QMnmAllowRenPot(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential" 
+QCapGenPlant(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
+QShaNewEqPowPla(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
+QDecisNewInv(allCy,YTIME)	               "Compute for Power Plant new investment decision"		
+QRenPotMaxAllow(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
```
Comment (giannou) [inline declarations.gms:159]: `QPotRenMaxAllow`

File: declarations.gms [hunk 14]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
+QRenPotMinAllow(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential"
```
Comment (giannou) [inline declarations.gms:160]: `QPotRenMinAllow`

File: declarations.gms [hunk 15]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QMaxmAllowRenPotent(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
-QMnmAllowRenPot(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential" 
+QCapGenPlant(allCy,PGALL,YTIME)	       "Compute electricity generation capacity"	
+QShaNewEqPowPla(allCy,PGALL,YTIME)	   "Compute the power plant share in new equipment"	
+QDecisNewInv(allCy,YTIME)	               "Compute for Power Plant new investment decision"		
+QRenPotMaxAllow(allCy,PGRENEF,YTIME)   "Compute maximum allowed renewable potential"
+QRenPotMinAllow(allCy,PGRENEF,YTIME)	   "Compute minimum allowed renewable potential" 
 QRenTechMatMult(allCy,PGALL,YTIME)	       "Compute renewable technologies maturity multiplier"	
-QElecProdChpPlants(allCy,CHP,YTIME)	       "Compute electricity production from CHP plants" 		 	
+QGenEstElecChpPlants(allCy,CHP,YTIME)	       "Compute electricity production from CHP plants"
```
Comment (giannou) [inline declarations.gms:162]: `QProdElecChp`

File: declarations.gms [hunk 16]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QTotElecGenCap(allCy,YTIME)	               "Compute total electricity generation capacity"
-QHourProdCostInv(allCy,PGALL,HOUR,YTIME)   "Compute hourly production cost used in investment decisions"
-QHourProdCostInvDec(allCy,PGALL,HOUR,YTIME)"Compute hourly production cost used in investment decisions"
-QGammaInCcsDecTree(allCy,YTIME)	           "Compute gamma parameter used in CCS/No CCS decision tree"
-qHourProdCostInvDecisionsAfterCCS(allCy,PGALL,HOUR,YTIME)	"Compute hourly production cost used in investment decisions taking into account CCS acceptance"
-QProdCostInvDecis(allCy,PGALL,YTIME)	            "Compute production cost used in investment decisions"
-QShrcapNoCcs(allCy,PGALL,YTIME)	                    "Compute SHRCAP excluding CCs"
-QProdCostTechPreReplac(allCy,PGALL,YTIME)	        "Compute production cost of technology  used in premature replacement"	
-QGapPowerGenCap(allCy,YTIME)	                    "Compute the gap in power generation capacity"												
+QCapGenElecNoChp(allCy,YTIME)	               "Compute total electricity generation capacity excluding CHP plants"
```
Comment (giannou) [inline declarations.gms:164]: `QCapElecNonChp`

File: declarations.gms [hunk 17]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
+QPotRenSuppCurve(allCy,PGRENEF,YTIME)	   "Compute renewable potential supply curve"		
+QIndexEndogScrap(allCy,PGALL,YTIME)	       "Compute endogenous scrapping index" 	
+QCostProdTechPreReplacAvail(allCy,PGALL,PGALL2,YTIME)	"Compute production cost of technology  used in premature replacement including plant availability rate"	
+QCostVarTechNotPGSCRN(allCy,PGALL,YTIME)   "Compute variable cost of technology excluding PGSCRN"	
+QCostVarTech(allCy,PGALL,YTIME)	           "Compute variable cost of technology" 	
+QProdReqTotElec(allCy,YTIME)               "Compute total required electricity production"
+QCapGenTotEstElec(allCy,YTIME)             "Compute Estimated total electricity generation capacity"	
+QCapGenTotElec(allCy,YTIME)	               "Compute total electricity generation capacity"
```
Comment (giannou) [inline declarations.gms:172]: `QCapElecTot`

File: declarations.gms [hunk 18]
```diff
@@ -118,152 +118,152 @@ iScaleEndogScrap(PGALL)                             "Scale parameter for endogen
...
-QElecPeakLoad(allCy,YTIME)	               "Compute elerctricity peak load"		
-QBslMaxmLoad(allCy,YTIME) 	               "Compute baseload corresponding to maximum load"
-QElecBaseLoad(allCy,YTIME)	               "Compute electricity base load"
-QShrcap(allCy,PGALL,YTIME)	               "Compute SHRCAP"	
-QElecGenCap(allCy,PGALL,YTIME)	           "Compute electricity generation capacity"
+QPeakLoad(allCy,YTIME)	               "Compute elerctricity peak load"		
+QBaseLoadMax(allCy,YTIME) 	               "Compute baseload corresponding to maximum load"
+QBaseLoad(allCy,YTIME)	               "Compute electricity base load"
+QShareNewTechCcs(allCy,PGALL,YTIME)	               "Compute SHRCAP"	
+QCapElec2(allCy,PGALL,YTIME)	           "Compute electricity generation capacity"
```
Comment (derevirn) [inline declarations.gms:130]: @Redmonkeycloud What's the meaning of 2 here?

**Comment by derevirn** [comment]
@giannou @Redmonkeycloud I noticed that camel case has been applied on some acronyms, e.g. Ccs and Co2. In my opinion, it's best practice to leave them in all caps. Let me know your thoughts on this.

**Comment by giannou** [comment]
> @giannou @Redmonkeycloud I noticed that camel case has been applied on some acronyms, e.g. Ccs and Co2. In my opinion, it's best practice to leave them in all caps. Let me know your thoughts on this.

I agree

**Review by derevirn — APPROVED** [review]

**Pull request closed without merging**